pkgs/servers/home-assistant/default.nix

```diff
@@ -799,6 +799,7 @@ in with py.pkgs; buildPythonApplication rec {
     tests = {
       inherit (nixosTests) home-assistant;
     };
+    pythonPackages = py.pkgs;
```

Comment:
```suggestion
    python = py;
```

Reply:
What would be the benefit of exposing all of it?

Reply:
I guess it's personal preference but we override `python`, not `pythonPackages`, so I think that should be the thing in `passthru`.

Reply:
WFM.